0.Preprocess_Data/scripts/4.aggregate_jump_data.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import gc
+import pathlib
+
+import pandas as pd
+import tqdm
+
+# In[ ]:
+
+
+# import jump data # absolute paths
+jump_data_path = pathlib.Path("/home/lippincm/Desktop/18TB/normalized_sc_data").resolve(
+    strict=True
+)
+
+# get the list of parquet files in the directory that are not aggregated
+jump_data_files = list(jump_data_path.glob("*.parquet"))
+jump_data_files = [x for x in jump_data_files if "agg" not in str(x)]
+df = pd.read_parquet(jump_data_files[0])
```

Comment:
If I am reading this correctly, you are only loading a non-aggregated dataset. If so, consider updating the comment above to indicate that you're only loading one.